src/main/java/io/cdap/plugin/gcp/bigquery/sink/BigQuerySinkUtils.java

```diff
@@ -48,6 +58,107 @@
   private static final String temporaryBucketFormat = gcsPathFormat + "/input/%s-%s";
   private static final String DATETIME = "DATETIME";
 
+  /**
+   * Creates Hadoop configuration for the given table and its fields.
+   *
+   * @param bucket bucket name
+   * @param tableName table name
+   * @param fields list of Big Query fields
+   * @return Hadoop configuration
+   */
+  public static Configuration getOutputConfiguration(Configuration baseConfiguration,
+                                                     BigQueryBaseConfig config,
+                                                     String bucket,
+                                                     String pathPrefix,
+                                                     String tableName,
+                                                     List<BigQueryTableFieldSchema> fields) throws IOException {
+    Configuration configuration = new Configuration(baseConfiguration);
+    String temporaryGcsPath = getTemporaryGcsPath(bucket, pathPrefix, tableName);
+
+    BigQuerySinkUtils.configureBigQueryOutput(configuration,
```

Comment:
So you need to qualify it here? It's same class

Reply:
Also I believe you can make configureBigQueryOutput private now (good as it takes mutable argument) and make naming consistent. We have `configureBigQueryOutput`, `configureBigQueryOutputForMultiSink`. The third method should have consistent naming

Reply:
Good catch on both.